runtime/src/main/scala/akka/grpc/javadsl/RouteUtils.scala

```diff
@@ -14,7 +14,7 @@ import akka.http.javadsl.model.{ HttpRequest, HttpResponse }
 import akka.http.javadsl.server.Route
 import akka.http.javadsl.server.directives.RouteAdapter
 import akka.http.scaladsl.server.RouteResult
-import akka.japi.Function
+import akka.japi.function.Function
```

Comment:
Instead of converting them to `akka.japi.function.Function`, I think we should not need these utilities at all anymore, we should be able to remove them here and replace them with helpers/directives now available in Akka HTTP